Gemfile

```diff
@@ -16,6 +16,8 @@ gem 'sequel-rails'
 gem 'pg'
 gem 'sequel-postgres-schemata', require: false
 
+gem 'aws-sdk-core'
```

Comment:
Not a fan of adding this to the Gemfile at the base level. Can we try to load the gems dynamically and disable the feature if they're not available? We can keep them in Gemfile in a separate group so that it's easier to test and deploy, while still possible to disable.